### PR TITLE
Added possibility to disable hand mesh and joint visualization

### PR DESCRIPTION
**Pull request by thalbern**
including buttons in sample scene to toggle hand joint and hand mesh

### CODE REVIEW

File: Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityHandTrackingProfile.cs [hunk 1]
```diff
@@ -47,7 +47,36 @@ public class MixedRealityHandTrackingProfile : BaseMixedRealityProfile
 
         [SerializeField]
         [Tooltip("If true and the hand mesh is available, try to access the hand mesh from the system. Note: this could reduce performance")]
-        private bool enableHandMeshUpdates = false;
-        public bool EnableHandMeshUpdates => enableHandMeshUpdates;
+        private bool enableHandMeshVisualization = false;
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityHandTrackingProfile.cs]: Might be good to add the [`FormerlySerializedAttribute`](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html
) here, so we don't break set-ups that already have this on.

File: Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityHandTrackingProfile.cs [hunk 2]
```diff
@@ -47,7 +48,37 @@ public class MixedRealityHandTrackingProfile : BaseMixedRealityProfile
 
         [SerializeField]
         [Tooltip("If true and the hand mesh is available, try to access the hand mesh from the system. Note: this could reduce performance")]
-        private bool enableHandMeshUpdates = false;
-        public bool EnableHandMeshUpdates => enableHandMeshUpdates;
+        [FormerlySerializedAs("EnableHandMeshUpdates")]
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityHandTrackingProfile.cs]:
```suggestion
        [FormerlySerializedAs("enableHandMeshUpdates")]
```

**Comment by cre8ivepark** [comment]
I'll merge with some of the updates in the dev branch and layout polish. Thanks!

**Comment by cre8ivepark** [comment]
Polished toggle button icons and layout.
2019-04-02 12_22_08-Unity 2018 3 8f1 Personal - HandInteractionExamples unity - MRTK-Public-Microsof

**Comment by wiwei** [comment]
Note that this should probably be going into mrtk_development instead of mrtk_development_hl2, now that the HL2 payload is in mrtk_development.

**Comment by davidkline-ms** [comment]
updated the target branch

**Pull request merged**